config.yaml

```diff
@@ -3,45 +3,84 @@ options:
     default: True
     type: boolean
     description: |
-      Limit requested identifiers (hostnames, IPs and OIDs) to first requester
```

Comment:
I think we should keep this as the first sentence of the description. It's a good summary. This is likely true of the other config options as well.

Reply:
Replaced the first sentences with the old shorter sentences.

This is harder than I thought :D